### ISSUE TITLE
Gene name mismatch with Ensembl

### ISSUE THREAD

**Issue by sksmi** [SACGF/variantgrid#249]
http://144.6.229.160/genes/view_gene_symbol/HSPA14

Search for gene symbol HSPA14 - VG maps symbol to 2 ids in Ensembl, not an issue in itself, just want to understand why ENSG00000284024 is mapped to HSPA14. 

GRCH38: 
[ENSG00000284024](http://144.6.229.160/genes/view_gene/ENSG00000284024) = AC069544.1 (ENSG) = LOC100421372 (RefSeq) =not in HGNC
Chromosome 10: 14,838,302-14,846,999 (+) 

[ENSG00000187522](http://144.6.229.160/genes/view_gene/ENSG00000187522) = HSPA14 (ENSG) = HSPA14 (RefSeq) = HSPA14 (HGNC)
Chromosome 10: 14,838,306-14,871,741 (+)

![image.png](https://images.zenhubusercontent.com/5bc5465086f3a678d497c0be/900c5468-da95-412e-8ee9-7028434331c5)

**Comment by davmlaw** [comment]
**TLDR:** The symbol assigned to a gene can change over time

If you go to the VG annotation page, you can see the last Ensembl annotation imported:

Homo_sapiens.GRCh38.100.gff3.gz: 227954 transcripts (Oct. 26, 2020, 2:06 p.m.) 

Here's the "gene" line from that GFF3 that match ENSG00000284024 - you can see it has HSPA14 on it

```
$ zgrep ENSG00000284024 Homo_sapiens.GRCh38.100.gff3.gz
10	ensembl_havana	gene	14838160	14847018	.	+	.	ID=gene:ENSG00000284024;Name=HSPA14;biotype=protein_coding;description=heat shock protein family A (Hsp70) member 14 [Source:NCBI gene%3BAcc:51182];gene_id=ENSG00000284024;logic_name=ensembl_havana_gene_homo_sapiens;version=2
```

If you look on the Ensembl site, they use v103. Downloading that file it contains:

```
$ zgrep ENSG00000284024 Homo_sapiens.GRCh38.103.gff3.gz
10	ensembl_havana	gene	14838302	14846999	.	+	.	ID=gene:ENSG00000284024;Name=AC069544.1;biotype=protein_coding;description=zinc finger and SCAN domain containing 29 [Source:NCBI gene (formerly Entrezgene)%3BAcc:100421372];gene_id=ENSG00000284024;logic_name=ensembl_havana_gene_homo_sapiens;version=4
```

That gene version went from version 2 to version 4 in less than a year....

**Comment by sksmi** [comment]
Thx.
